src/Native/System.Security.Cryptography.Native/evp.cpp

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Also, another pattern we have in the shims is to have a header documenting the public interface. I actually went back and forth on this and considered just having .cpp since we only P/Invoke, but when I discussed it last with @socket, he felt there was value in keeping the headers, so we stuck with it.